app/views/searches/show.html.erb

```diff
@@ -7,6 +7,17 @@
   </div>
 <% end %>
 
+<% if @gems && @gems.size < 1 && @gems.respond_to?(:response) && (suggestions = @gems.response.response['suggest']) && suggestions.present? && suggestions['suggest_name'].present? && suggestions['suggest_name'][0]['options'].length > 0  %>
```

Comment:
seems like a situation tailor-made for hash/array dig or the new safe navigation operator. Helper for now seems more readable at least.